CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-*       @sownak @suvajit-sarkar @adityajoshi12
+*       @sownak @suvajit-sarkar @adityajoshi12 @alvaropicazo

```

Comment:
Alvaro has to be added to the groups first.

Reply:
@sownak what group? I invited @alvaropicazo to https://github.com/orgs/hyperledger-bevel/teams/bevel-committers. @alvaropicazo kindly accept the invite. Is that the group ?

Reply:
He is getting auto removed. I think there was a page where we have to add his id and commit it, I'm not able find the link. @ryjones would be able to help us with this?